File: quan.cs/Test/Config/Item/UseEffect2.cs
using Newtonsoft.Json.Linq;
using Quan.Utils;

namespace Test.Config.Item
{
    /// <summary>
	/// 使用效果2<br/>
	/// 代码自动生成，请勿手动修改
	/// </summary>
    public class UseEffect2 : UseEffect
    {
        public readonly int bbb;


        public UseEffect2(JObject json) : base(json)
        {
            bbb = json["bbb"]?.Value<int>() ?? default;
        }

        public new static UseEffect2 Create(JObject json)
        {
            return new UseEffect2(json);
        }

        public override string ToString()
        {
            return "UseEffect2{" +
                   "aaa=" + aaa.ToString2() +
                   ",bbb=" + bbb.ToString2() +
                   '}';
        }
    }
}

File: quan.java/quan-config/src/test/java/quan/config/item/UseEffect2.java
package quan.config.item;

import com.alibaba.fastjson.*;

/**
 * 使用效果2<br/>
 * 代码自动生成，请勿手动修改
 */
public class UseEffect2 extends UseEffect {

    public final int bbb;


    public UseEffect2(JSONObject json) {
        super(json);

        this.bbb = json.getIntValue("bbb");
    }

    public static UseEffect2 create(JSONObject json) {
        return new UseEffect2(json);
    }

    @Override
    public String toString() {
        return "UseEffect2{" +
                "aaa=" + aaa +
                ",bbb=" + bbb +
                '}';

    }

}
